lab_1_keywords_tfidf/main.py

```diff
@@ -156,4 +256,21 @@ def extract_significant_words(chi_values: dict[str, float], alpha: float) -> Opt
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    # есть ощущение что я неправильно реализовала функцию
+    # нужно брать альфу из критериона, это есть в чате в тг
+    criterion = {0.05: 3.842, 0.01: 6.635, 0.001: 10.828}
+    crit_keys = list(criterion.keys())
+    if alpha not in crit_keys:
+        return None
+    if not chi_values or not alpha:
+        return None
+    if not isinstance(chi_values, dict) or not isinstance(alpha, float):
```

Comment:
`not isinstance(alpha, float):` is a redundant check given the check in line 263

Reply:
exactly so